ENGAGEHF/SharedContext/FHIRSystem.swift

```diff
@@ -0,0 +1,21 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+
+
+enum FHIRSystem {
+    case loinc
+    
+    
+    var url: URL {
+        switch self {
+        case .loinc: URL(string: "http://loinc.org")!
+        }
+    }
+}
```

Comment:
```suggestion
enum FHIRSystem {
    // swiftlint-disable-next force-unwrapping
    static let loinc = URL(string: "http://loinc.org")!
}
```

What do you think about this instead?